src/Compiler/CompilerRS.cpp

```diff
@@ -95,6 +95,8 @@ plugin -i ${PLUGIN_LIB}
 
 ${PLUGIN_NAME} -tech ${MAP_TO_TECHNOLOGY} ${OPTIMIZATION} ${EFFORT} ${CARRY} ${IO} ${KEEP_TRIBUF} ${NEW_DSP19X2} ${NEW_TDP36K} ${LIMITS} ${FSM_ENCODING} ${FAST} ${NO_FLATTEN} ${MAX_THREADS} ${NO_SIMPLIFY} ${CLKE_STRATEGY} ${CEC}
 
+plugin -i design-edit
```

Comment:
The same code has to be replicated in 3 more places:

Lines 114 130 152

Reply:
Preserve the original gate level file for Simulation purpose, rename the fabric file:
fabric_<project>.....

Reply:
No, we simply change the file name that goes in vpr. I'm doing it now.